src/components/Modals/DeleteClientModal.jsx

```diff
@@ -30,35 +32,28 @@ const DeleteClientModal = ({
   setShowDeleteClientModal,
   selectedClientToDelete
 }) => {
+  const notify = useNotification();
   const { state, dispatch } = useStatusNotification();
   const { removeUser } = useContext(UserListContext);
-
+  const [showSpinner, setShowSpinner] = useState(false);
   // Event handler for deleting client from client list
   const handleDeleteClient = async (event) => {
     event.preventDefault();
-    runNotification(
-      `Deleting "${selectedClientToDelete?.person}" from client list...`,
-      5,
-      state,
-      dispatch
-    );
+    setShowSpinner(true);
+    // notify.addNotification('info',`Deleting "${selectedClientToDelete?.person}" from client list...`)
     try {
       await removeUser(selectedClientToDelete);
     } catch (e) {
       runNotification(`Client deletion failed. Reason: ${e.message}`);
     } finally {
-      runNotification(
-        `"${selectedClientToDelete?.person}" deleted from client list...`,
-        5,
-        state,
-        dispatch
-      );
+      notify.addNotification('success',`"${selectedClientToDelete?.person}" deleted from client list...`)
```

Comment:
The `deleted` notification has an ellipsis at the end whereas the `added` one does not. Not vital but I'd personally like to have it be consistent. I prefer the version without an ellipsis.

Reply:
I pushed this and then noticed it wasn't up to date, then pulled from Development and I have merger conflicts. I'm going to try and resolve in the morning.

Reply:
Alright. I'll postpone reviewing until that's ready.